console/src/components/Common/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Toggle from 'react-toggle';
+import './Toggle.css';
+import 'react-toggle/style.css';
+
+type ToggleProps = {
+  onChange: () => void;
+  icons: boolean;
+  checked: boolean;
+};
```

Comment:
These are not new types, but part of ReactTogle lib. 
Install `@types/react-toggle` and get type of Props from there.  
They are exported as ToggleProps (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-toggle/index.d.ts).